vizro-core/examples/kpi/utils/_helper.py

```diff
@@ -0,0 +1,67 @@
+"""Contains helper functions."""
+
+import numpy as np
+import pandas as pd
+
+REGION_MAPPING = {
```

Comment:
This could probably be done a bit more nicely with something like
```
REGION_MAPPING = {**dict.fromkeys(["CT", "ME", "MA", "NH", "RI", "VT", "NJ", "NY", "PA"], "North East"), **...}
```